02week/pigLatin.js

```diff
@@ -9,8 +9,41 @@ const rl = readline.createInterface({
 
 
 function pigLatin(word) {
+  let piggy = word.toLowerCase().trim();
 
-  // Your code here
+  if (piggy[0]==='a'||piggy[0]==='e'||piggy[0]==='i'||piggy[0]==='o'||piggy[0]==='u'){
```

Comment:
while this works for words that have a vowel in the first 6 letters, it is not generic enough to work for all words.
It also does not handle words with no vowels like "fry".

In the general case, you would can use ```piggy.indexOf('a')```, ```piggy.indexOf('e')```, etc ... to see where the vowel are, and pick the smallest one and use that with ```piggy.substring```